### DISCUSSION TITLE
menu editing

### DISCUSSION THREAD

**Discussion by Marcioghiraldelli** [AzuraCast/AzuraCast#6518] [General]
I like to modify the azuracast menu manually, leaving the writing to my understanding, even putting a few more links.
However, in this last stable version, the location of the station.php menu file was removed and now I can't find the location of the new menu file.
Can anyone give me a light on how to make this change again? or pass me the way?
thanks in advance

**Comment by BusterNeece** [comment]
The corresponding files for the Admin and Station menus are now located here:

Admin: https://github.com/AzuraCast/AzuraCast/blob/main/frontend/vue/components/Admin/menu.ts
Stations: https://github.com/AzuraCast/AzuraCast/blob/main/frontend/vue/components/Stations/menu.ts

Because they're frontend Vue code, if you're on production mode you'll have to rebuild the frontend code (`npm run build` in the frontend folder) to see changes take effect. This isn't necessary on development mode.
